astropy/utils/shapes.py

```diff
@@ -57,14 +70,16 @@ class NDArrayShapeMethods:
     def __getitem__(self, item):
         return self._apply("__getitem__", item)
 
-    def copy(self, *args, **kwargs):
+    def copy(self, order: Literal["C", "F", "A", "K"] = "C") -> Self:
         """Return an instance containing copies of the internal data.
 
         Parameters are as for :meth:`~numpy.ndarray.copy`.
         """
-        return self._apply("copy", *args, **kwargs)
+        return self._apply("copy", order=order)
 
-    def reshape(self, *args, **kwargs):
+    def reshape(
+        self, *newshape: int | Sequence[int], order: Literal["C", "F", "A"] = "C"
```

Comment:
The suggested code would break at least https://github.com/astropy/astropy/blob/16786add79acbd89743b006d5eccf7d270089047/astropy/coordinates/tests/test_erfa_astrom.py#L66

Reply:
You're right. I think I got confused while reading numpy docs. `order` is in fact *not* allowed a positional.